app/styles/_library/_objects.text.scss

```diff
@@ -78,19 +78,19 @@
   /*font-weight: normal;*/
 }
 .o-rte-text > h1 {
-  @include typeface(header, 5);
+  @include typeface(header, 11);
   @include media(">large") {
-    @include typeface(header, 9);
+    @include typeface(header, 15);
```

Comment:
Just a note that this feels weird because the typeface mixin includes media queries itself, so we're creating nested media queries here...
I think maybe we should just set the size token in the media selector here...



Also suddenly feeling doubt that the mixin should be automatically applying mobile size adjustments universally. (might be the cause of some of our weird mobile sizing issues) Lastly feeling like it's weird that the mixin isn't doing mobile first font sizing and then apply the larger styles for larger displays.  Anyway, probably something to discuss later rather than address in this PR.

Reply:
looks like the mixin handles `<medium`, whereas this selector handles `>large`.

are you saying to remove the `<medium` selector from the mixin? it seems like that would have side effects.